internal/data/generated_metrics.go

```diff
@@ -986,13 +965,6 @@ func (ms HistogramDataPoint) LabelsMap() StringMap {
 	return newStringMap(&(*ms.orig).Labels)
 }
 
-// SetLabelsMap replaces the Labels associated with this HistogramDataPoint.
-//
-// Important: This causes a runtime error if IsNil() returns "true".
-func (ms HistogramDataPoint) SetLabelsMap(v StringMap) {
-	(*ms.orig).Labels = *v.orig
-}
-
```

Comment:
By removing this, we will not have an option to use pre-built `StringMap` anymore, right? Here we would need HistogramDataPoint instance to be ready first. I think, having pre-built StringMap may be helpful, for example I use that approach in https://github.com/open-telemetry/opentelemetry-collector/pull/742

Reply:
Correct, the reason is here https://github.com/open-telemetry/opentelemetry-collector/issues/653. If we offer set we need to do a deep copy of that object to ensure the user does not set the same labels to multiple points or does not set the same lables to different data.

Reply:
That makes sense. That's the mistake I made in https://github.com/open-telemetry/opentelemetry-collector/pull/763 as well